core/common/src/ITwinError.ts

```diff
@@ -23,21 +24,44 @@ export interface InUseLock {
    */
   state: LockState;
   /** An array of Briefcase ids that hold this lock. */
-  briefcaseIds: number[];
+  briefcaseIds: BriefcaseId[];
 }
 
+/**
```

Comment:
I realize that this PR has already merged, but this is not the right place and not the right name for these channel error types. 

The type `iTwinErrorMessages` is definitely not a "record for all itwin error messages". Maybe it is a place for all iTwin errors thrown by "core-backend"? I'd move it to a file called "ChannelErrors" or something so it doesn't encourage other to think they need to add entries here.